busy_beaver/apps/web/forms.py

```diff
@@ -10,12 +10,31 @@
 TZ_CHOICES = sorted(
     add_gmt_offset_to_timezone(TIMEZONES), key=lambda x: int(x[1][3:5]), reverse=True
 )
+WEEKDAYS = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+]
 
 
 class GitHubSummaryConfigurationForm(FlaskForm):
+    channel = SelectField(label="Channel")
+
     summary_post_time = TimeField(
         "When to post GitHub Summary?", validators=[DataRequired()]
     )
     summary_post_timezone = SelectField(
         label="Timezone", choices=TZ_CHOICES, default="UTC"
     )
+
+
+class UpcomingEventsConfigurationForm(FlaskForm):
+    channel = SelectField(label="Channel")
+
+    day_of_week = SelectField("Day to post", choices=WEEKDAYS, default="Monday")
+    post_time = TimeField("Time to post", validators=[DataRequired()])
+    post_timezone = SelectField(label="Timezone", choices=TZ_CHOICES, default="UTC")
```

Comment:
Originally started working on the Upcoming Events configuration, but it makes sense to simplify onboarding before doing that work